test/integration/tests/4045-no-run-benchmarks/files/foff-bench-not-build.cabal

```diff
@@ -0,0 +1,72 @@
+-- This file has been generated from package.yaml by hpack version 0.28.2.
```

Comment:
Let's keep the cabal file in the repo, without superfluous comments and delete the hpack file. This ensures consistency with the other test suites.